site/src/main/webapp/WEB-INF/jsp/errorpages/ErrorPage404.jsp

```diff
@@ -2,18 +2,14 @@
 <%@ include file="/WEB-INF/jspf/htmlTags.jspf" %>
 <%@ page isErrorPage="true" %>
 <% response.setStatus(404); %>
+
 <html lang="en">
 <head>
   <meta charset="utf-8"/>
   <title>404 error</title>
 </head>
 <body>
-<h2>Welcome to Hippo</h2>
-<p>
-  It appears that you just created an empty Hippo project from the archetype. There is nothing to show on the site yet.
-  We recommend you use
-  <a href="http://<%=request.getServerName() + ':' + request.getServerPort() + "/essentials"%>" target="_blank">Hippo's setup application</a>
-  to start developing your project.
-</p>
+<h1>Page not found!!!</h1>
```

Comment:
How about leaving the exclamation marks off - the users will be stressed out enough as it is, by breaking our website ;-)

Reply:
Will remove...we don't want user stress :-)  The 404 page was auto added when the essential search plugin was added to the project.  We can revisit these default error pages once styling etc has been defined.